.github/actions/run-sql-script/action.yml

```diff
@@ -0,0 +1,49 @@
+name: 'run-sql-script'
+description: 'Run a script against a CloudSQL instance'
+author: 'dd'
+inputs:
+  db-connection-name:
+    description: 'full CloudSQL connection name: <project>:<region>:<cloudsql-instance>'
+    required: true
+  db-name:
```

Comment:
Might be good to put a guard up so that it doesn't run against prod unless `damnTheTorpedoes` is also true, or something.